### PR TITLE
complete implementation of vpc endpoint

### PR DESCRIPTION
**Pull request by wuebbels**

### CODE REVIEW

**Comment by anton-sidelnikov** [comment]
Hi @wuebbels , do not forget to add release note by reno(https://pypi.org/project/reno/)

File: otcextensions/sdk/vpcep/v1/endpoint.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    #: Domain status.
+    active_status = resource.Body('active_status', type=list)
+    #: Creation time of the VPC endpoint.
+    created_at = resource.Body('created_at')
+    #: Description of the VPC endpoint.
+    description = resource.Body('description')
+    #: Domain name for accessing the associated VPC endpoint service.
+    dns_names = resource.Body('dns_names', type=list)
+    #: Whether the VPC endpoint is enabled.
+    enable_status = resource.Body('enable_status')
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/vpcep/v1/endpoint.py]: is_enabled

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/vpcep/v1/endpoint.py (hunk 1 above)
Comment (vineet-pruthi) [reply otcextensions/sdk/vpcep/v1/endpoint.py]: Hi @anton-sidelnikov , its string variable  ,"enable_status":"enable",  thats why i didnt change the name to is_enabled

Comment (anton-sidelnikov) [reply otcextensions/sdk/vpcep/v1/endpoint.py]: then better to call it simple `status`, and rename also status from line 86 to `connection_status`. Weird naming in API

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/vpcep/v1/endpoint.py (hunk 1 above)
Comment (vineet-pruthi) [reply otcextensions/sdk/vpcep/v1/endpoint.py]: updated `enable_status` to `is_enabled` (type=bool)

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**